rasa/cli/x.py

```diff
@@ -220,67 +226,38 @@ def is_rasa_project_setup(project_path: Text):
     return True
 
 
-def rasa_x(args: argparse.Namespace):
-    from rasa.cli.utils import print_success, print_error, signal_handler
-    from rasa.core.utils import AvailableEndpoints
+def _validate_rasa_x_start(args: argparse.Namespace, project_path: Text):
+    if not is_rasa_x_installed():
+        print_error(
+            "Rasa X is not installed. The `rasa x` "
+            "command requires an installation of Rasa X."
```

Comment:
while we are here: can we add the installation instructions here, or at least a link to the docs?